ci-operator/jobs/konveyor/mig-operator/konveyor-mig-operator-master-presubmits.yaml

```diff
@@ -7,6 +7,7 @@ presubmits:
     cluster: build01
     context: ci/prow/ci-index
     decorate: true
+    skip_if_only_changed: "^docs/|\.(md|adoc)$|^(README|LICENSE)$"
```

Comment:
doesn't this say only run when docs changed?

Reply:
I dont think so, The document says the following.
```
skip_if_only_changed skips the job if all paths match. For example, you may wish to skip a compilation job for pull requests that only change documentation files:

presubmits:
  org/repo:
  - name: compile-job
    always_run: false
    skip_if_only_changed: "^docs/|\.(md|adoc)$|^(README|LICENSE)$"
    
    
Both of the above examples would trigger on a pull request containing foo/bar.c and SECURITY.md, but not one containing only SECURITY.md.

```
So a PR raised only with doc change should not trigger anything in this case.

Reply:
I wonder if using a single-quoted string will work better?